cmd/osbuild-worker/main.go

```diff
@@ -8,14 +8,14 @@ import (
 	"flag"
 	"fmt"
 	"io/ioutil"
-	"log"
 	"os"
 	"os/exec"
 	"path"
 	"strings"
 	"time"
 
 	"github.com/BurntSushi/toml"
+	log "github.com/sirupsen/logrus"
```

Comment:
Is there a good reason to call this log instead of logrus? All the other files are using logrus so it could be confusing.

Reply:
Good point, I'll fix it, thanks